mypy/build.py

```diff
@@ -353,7 +353,7 @@ def __init__(self, data_dir: str,
         self.semantic_analyzer = SemanticAnalyzer(lib_path, self.errors)
         self.modules = self.semantic_analyzer.modules
         self.semantic_analyzer_pass3 = ThirdPass(self.modules, self.errors)
-        self.type_checker = TypeChecker(self.errors, self.modules)
+        self.type_checker = TypeChecker(self.errors, self.modules, options.python_version)
```

Comment:
You shouldn't need this -- you can access the version already via self.options.python_version.